.github/actions/build_ci/entrypoint.sh

```diff
@@ -74,8 +74,13 @@ build_zephyr(){
     	cd ./zephyr &&
     	source zephyr-env.sh &&
 	cd ../.. &&
-	cmake . -DWITH_ZEPHYR=on -DBOARD=qemu_cortex_m3 -DWITH_TESTS=on -Bbuild-zephyr &&
-	cd build-zephyr &&
+        echo  "###### Build for qemu_cortex_m3 ######" &&
+        cmake . -DWITH_ZEPHYR=on -DBOARD=qemu_cortex_m3 -DWITH_TESTS=on -Bbuild-zephyr-m3 &&
+        cd build-zephyr-m3 &&
+        make VERBOSE=1 &&
+        echo  "###### Build for qemu_xtensa ######" &&
```

Comment:
```suggestion
        make VERBOSE=1 &&
        cd .. &&
        echo  "###### Build for qemu_xtensa ######" &&
```

Reply:
@carlocaione thanks. Aside from missing `cd .. &&` I don't understand the other diff. Is it because I used wrong coding style? (tabs, etc?)

Reply:
haa nope, it's just that I selected three lines for context and github is not smart enough to figure out the correct diff ;) yeah, it's just the `cd ..`